go.mod

```diff
@@ -3,16 +3,14 @@ module github.com/canonical/pebble
 go 1.14
 
 require (
+	github.com/canonical/x-go v0.0.0-20230113154138-0ccdb0b57a43
 	github.com/gorilla/mux v1.8.0
 	github.com/gorilla/websocket v1.4.2
 	github.com/jessevdk/go-flags v1.4.0
-	github.com/kr/text v0.2.0 // indirect
 	github.com/pkg/term v1.1.0
 	golang.org/x/crypto v0.0.0-20201221181555-eec23a3978ad
-	golang.org/x/net v0.0.0-20210119194325-5f4716e94777 // indirect
 	golang.org/x/sys v0.0.0-20210119212857-b64e53b001e4
 	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c
 	gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
-	gopkg.in/yaml.v2 v2.4.0
-	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
+	gopkg.in/yaml.v3 v3.0.1
```

Comment:
Was it intentional to update `yaml.v3` from v3.0.0 to v3.0.1? Sounds good to me, just curious.

Reply:
Includes @niemeyer 's The Billion Dollar Mistake bugfix :)